### DISCUSSION TITLE
早上看新闻，官方uwp倒了，感觉以后xbox用户应该会越来越多，要么用浏览器要么第三方

### DISCUSSION THREAD

**Discussion by zhulibo** [Richasy/Bili.Uwp#1206] [常规]
下班回去试试这个软件，希望可以对xbox多做些优化

**Comment by 008exe** [comment]
悄悄用吧，xbox没办法像pc那样容易侧载，只能商店安装，要是叔叔再来一封律师函就要下架了

**Reply by marcosteam** [reply]
> 

Github 不抗 DMCA，最好的办法就是把源码放在自己的服务器上（参考 unlock-music）

**Comment by xiaowu2001** [comment]
官方uwp听说是微软下架的，官方声称会尽快恢复。原因未知，有可能是因为那个uwp能白嫖HEVC扩展。
